src/main/java/org/gridsuite/modification/server/repositories/ModificationRepository.java

```diff
@@ -0,0 +1,26 @@
+/*
+  Copyright (c) 2021, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.repositories;
+
+import java.util.List;
+import java.util.UUID;
+
+import org.gridsuite.modification.server.entities.ModificationEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+/**
+ * @author Slimane Amar <slimane.amar at rte-france.com>
+ */
+@Repository
+public interface ModificationRepository extends JpaRepository<ModificationEntity, UUID> {
+    @Query(value = "SELECT *, 0 AS clazz_ FROM modification WHERE group_uuid = ?1", nativeQuery = true)
+    List<ModificationEntity> findAll(UUID uuid);
```

Comment:
rename to a more explicit name saying that it is not a polymorphic query (only the base data is retrieved). Do we really need this ? If we do a DTO projection on the base class fields only, does it still do the joins ?

Reply:
or can we remove inheritance and replace it with just relations ? For exemple ElementaryModificationEntity would have only the data regarding the actual modification, and a one to one mapping to a Modification which has the generic data.

This would still allow to select the base attribute like in this query, but not allow to select both elementaryModifications values and other modification specific values.

Reply:
Yes, we need to get general information about group modifications  like convergence
I tested with a base DTO class and JPA still do joins

Reply:
> rename to a more explicit name saying that it is not a polymorphic query 

findAllBases ?
findAllBasesByGroupUuid ?

Reply:
Done (renaming)